make_tools.mk

```diff
@@ -0,0 +1,24 @@
+# This controls building executables in the `tools` folder.
+# Can be invoked through the `Makefile` or standalone.
+
+MAKEFLAGS += --no-print-directory
+
+# Inclusive list. If you don't want a tool to be built, don't add it here.
+# Use path relative to this makefile, in order fo Makefiles in subdirectory to work
+MKFILE_PATH := $(abspath $(lastword $(MAKEFILE_LIST)))
+TOOLS_DIR := $(dir $(MKFILE_PATH))/tools
+TOOL_NAMES := aif2pcm bin2c gbafix gbagfx mid2agb preproc scaninc
+
+TOOLDIRS := $(TOOL_NAMES:%=$(TOOLS_DIR)/%)
+
+# Tool making doesnt require a pokeemerald dependency scan.
```

Comment:
Doesn't make sense in this repo

Reply:
If you mean it has the wrong name, done!